### DISCUSSION TITLE
Connecting with ELCO system

### DISCUSSION THREAD

**Discussion by AlbertoB11** [john30/ebusd#730] [Q&A]
I'm going to interface with ELCO system bus. 
I readed ariston, elco and chaffoteaux are the same factory. Looking at manual it seems the same electronics. 
My system is a hybrid gas/heatpump for home heating. 
Because i'm not 100% sure in connecting ebusd interface to my system bus, because of electrical compatibility, i don't want to destroy all boards. 
Can i make some test before connecting ebusd electrical interface to elco system bus ?

**Comment by wrongisthenewright** [comment]
Ciao,
take a look at my repo, is a custom configuration for Ariston/Elco/Chaffoteaux ebus "dialect":

https://github.com/wrongisthenewright/ebusd-configuration-ariston-bridgenet

for the elctrical part you can access the bus, in parallel with other device on one of the many different ports available:
- boiler
- hp
- hybrid manager
- remote control

how to do it depends on what kind of adapter you are using.
